### DISCUSSION TITLE
Text translation in Spanish is of bad quality

### DISCUSSION THREAD

**Discussion by cafeina-software** [fwupd/fwupd#7462] [General]
**Describe the question**
I don't know if here are handled translations, but the text translations in Spanish are of bad quality, utterly garbage. These include non translated lines, bad translations (or translations by a non-native speaker) and bad adaptations. Examples from the _security_ command:

`✔ Los PCR vacíos de TPM:         Válido`
No good translation would use the article "los" here. Also the field name sounds somewhat strange.

`✔ BIOS firmware updates:         Activado`
Missing translation.

`✘ Suspendido-a-descanso:         Deshabilitado`
Trashy translation without proper adaptation. This "dashy" thing to link words to create a new "word" is not used in Spanish, or at least not used in the current Spanish.

`✘ Procesador retira protección:  Deshabilitado`
Hard to understand field name under its context. 

**fwupd version information**
Please provide the version of the daemon and client if applicable to your current installation of `fwupd`.

```shell
compile   org.freedesktop.fwupd         1.9.21
compile   org.freedesktop.Passim        0.1.8
compile   com.hughsie.libxmlb           0.3.19
compile   com.hughsie.libjcat           0.2.1
runtime   org.freedesktop.fwupd-efi     1.6
compile   org.freedesktop.gusb          0.4.9
runtime   com.hughsie.libxmlb           0.3.19
runtime   com.hughsie.libjcat           0.2.1
runtime   org.freedesktop.gusb          0.4.9
runtime   org.freedesktop.fwupd         1.9.21
runtime   org.kernel                    6.9.7-arch1-1
```

**Comment by hughsie** [comment]
Corrections most welcome, thanks! https://app.transifex.com/freedesktop/fwupd/
